commcare_connect/opportunity/visit_import.py

```diff
@@ -290,14 +285,40 @@ def get_exchange_rate(currency_code, date=None):
     if currency_code == "USD":
         return 1
 
+    rate = get_exchange_rate_from_db(currency_code, date)
```

Comment:
It may not be worth to make this a separate method as it's just a queryset call.